modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -601,82 +601,6 @@ definition {
 			actionName = "Action Name 2");
 	}
 
-	@description = "LPS-145665 - Verify that Action can be triggered after disabling the expression"
-	@priority = "4"
-	test CanTriggerActionAfterDisablingExpression {
-		task ("Add an object, field and action") {
-			ObjectAdmin.addObjectViaAPI(
-				labelName = "Custom Object",
-				objectName = "CustomObject",
-				pluralLabelName = "Custom Objects");
-
-			ObjectAdmin.addObjectFieldViaAPI(
-				fieldBusinessType = "Text",
-				fieldLabelName = "Custom Field",
-				fieldName = "customObjectField",
-				fieldType = "String",
-				isRequired = "true",
-				objectName = "CustomObject");
-
```

Comment:
Hey it looks like you added this test by mistake in [LPS-156343 - Requested Modifications and fix some tests](https://github.com/timpak/liferay-portal/pull/309/commits/2cc5cd9e4a2ca625674b2d12a499e28cb9de44e7). Instead of making a separate commit, can you fix this in the original commit?